src/Plugin/GraphQL/DataProducer/Entity/Fields/Image/ImageDerivative.php

```diff
@@ -103,7 +103,7 @@ public function resolve(FileInterface $entity = NULL, $style, RefinableCacheable
       // @phpstan-ignore-next-line
       $height = $entity->height;
 
-      if (empty($width) || empty($height)) {
+      if (!isset($width) || !isset($height)) {
```

Comment:
@apathak18 We've updated condition from `empty` to `!isset`. The empty condition check, returns `TRUE` for empty string value and returns `FALSE` for `NULL` value, whereas `isset` return `TRUE` for `NULL` value and return `FALSE` for empty string value. If we want to limit condition only for `NULL` value, then better we should update it `$width == NULL` for better code visibility.